src/systems/filecoin_mining/storage_mining/storage_mining_subsystem.go

```diff
@@ -289,42 +286,43 @@ func (sms *StorageMiningSubsystem_I) VerifyElectionPoSt(inds indices.Indices, he
 
 	postCfg := filproofs.ElectionPoStCfg(sectorSize)
 
-	pvInfo := sector.PoStVerifyInfo_I{
-		OnChain_:    onChainInfo,
-		Randomness_: onChainInfo.Randomness(),
+	pvInfo := abi.PoStVerifyInfo{
```

Comment:
some fields are missing here, perhaps optional

Reply:
I'm not trying to fix anything here. There are many things missing all over the spec.